tox.ini

```diff
@@ -65,16 +65,17 @@ commands =
     buildhtml: bash -c 'cat ignore_tutorials/ignore_rendering_execution >> ignore_testing'
 
     # We only want to run CI in PRs for the notebooks we touched
-    !buildhtml: bash -c 'if [[ $GITHUB_EVENT_NAME == pull_request && -z "$(grep force_run:all_tests ${GITHUB_EVENT_PATH})" ]]; then git fetch origin main --depth=1; git diff origin/main --name-only tutorials | grep .md; else find tutorials -name "*.md"; fi | grep -vf ignore_testing/ignore_testing | xargs jupytext --to notebook '
+    !buildhtml: bash -c 'if [[ $GITHUB_EVENT_NAME == pull_request && -z "$(grep force_run:all_tests ${GITHUB_EVENT_PATH})" ]]; then git fetch origin main --depth=1; git diff origin/main --name-only tutorials | grep .md; else find tutorials -name "*.md"; fi | grep -vf ignore_testing | xargs jupytext --to notebook '
 
+    !buildhtml: bash -c "echo 'Notebooks ignored (not tested/executed) in this job:\n'; cat ignore_testing"
     !buildhtml: pytest --nbval-lax -vv --suppress-no-test-exit-code --durations=10 tutorials
 
     # sed -i needs a bit of hacky conditional on ubuntu to cover the case of an empty ignore
     buildhtml: bash -c "find tutorials -name '*md' | grep -f ignore_testing | sort | uniq > ignore_execute; if [ -s ignore_execute ]; then cat ignore_execute | xargs -n 1 sed -i -e 's|name: python3|name: python3\nskip_execution: true|g';fi"
 
-    # Status check is a workaround until https://github.com/jupyter-book/mystmd/issues/2113 is fixed
-    buildhtml: bash -c "jupyter-book build --execute --html 2>&1 | tee /tmp/mystbuild.log"
-    buildhtml: bash -c "if grep -q 'Traceback .most recent call last.' /tmp/mystbuild.log; then exit 1; fi"
+    buildhtml: bash -c "echo 'Notebooks ignored (not tested/executed) in this job:\n'; cat ignore_execute"
+    # Using srtict so we fail with trackbacks and debug mode to have a richer log
+    buildhtml: bash -c "npx myst build --execute --html --strict -d"
```

Comment:
The publish job failed after this merged. https://github.com/Caltech-IPAC/irsa-tutorials/actions/runs/18979533957/job/54208233775. It's complaining that it can't find the myst executable when it tries to run line 78. Maybe it's not being installed?